src/components/Weather.js

```diff
@@ -0,0 +1,37 @@
+import Radium from 'radium';
+import React from 'react';
+import * as WeatherActions from '../Redux/Modules/WeatherReducer';
+
+class Weather extends React.Component {
+  componentDidMount () {
+    WeatherActions.getWeather();
+    setTimeout(() => {
+      this.forceUpdate();
+    }, 1000);
+  }
```

Comment:
I don't like the way I'm doing this. currently, i get the weather and update the state, wait 1 second then rerender the page using the new state. This is because async is a thing. Anyone have any suggestions to make this less ghetto?